pyproject.toml

```diff
@@ -94,23 +94,16 @@ filterwarnings = [
 ]
 
 [tool.ruff]
-select = [
-    "E",
-    "F", # flake8
+src = ["src"]
+lint.extend-select = [
```

Comment:
E and F (with a few useful exclusions that conflict with formatters) are in the default set, so extend-select is better.